crates/ruff_python_formatter/src/module/mod_module.rs

```diff
@@ -1,12 +1,12 @@
-use crate::{FormatNodeRule, PyFormatter};
+use crate::{range_verbatim, FormatNodeRule, PyFormatter};
 use ruff_formatter::FormatResult;
 use rustpython_parser::ast::ModModule;
 
 #[derive(Default)]
 pub(crate) struct FormatModModule;
 
 impl FormatNodeRule<ModModule> for FormatModModule {
-    fn fmt_fields(&self, _item: &ModModule, _f: &mut PyFormatter) -> FormatResult<()> {
-        Ok(())
+    fn fmt_fields(&self, item: &ModModule, f: &mut PyFormatter) -> FormatResult<()> {
+        range_verbatim(item.range, f)
```

Comment:
The `FormatModModule` should be called by our integration tests 

https://github.com/charliermarsh/ruff/blob/a7b8f87c4099bed61ddb11112a8410a68ca6fdb5/crates/ruff_python_formatter/src/lib.rs#L92-L181